src/pods/movies/components/filters/textFieldForm.tsx

```diff
@@ -0,0 +1,44 @@
+import * as React from "react";
+import TextField from "@material-ui/core/TextField";
+import Typography from "@material-ui/core/Typography/Typography";
+
+interface Props {
+  name: string;
+  label: string;
+  onChange: any;
+  value: string;
+  error?: string;
+  type? : string;
+  className?: string;
+}
+
+const defaultProps : Partial<Props> = {
+  type: 'text', 
+}
```

Comment:
We are not using this default props, i nthis code we hhave to check how it works

Reply:
Try this: https://stackoverflow.com/questions/37262047/react-with-typescript-define-defaultprops-in-stateless-function